src/ucp/core/ucp_rkey.h

```diff
@@ -43,7 +43,6 @@ enum {
 struct ucp_rkey_config_key {
     ucp_md_map_t                  md_map;       /* Which *remote* MDs have valid memory handles */
     ucp_worker_cfg_index_t        ep_cfg_index; /* Endpoint configuration */
-    ucs_sys_device_t              sys_dev;      /* Remote device id */
```

Comment:
What was the original point of having sys_dev in rkey?

Reply:
it was a mistake since the remote device index has no meaning in the local scope